install_config/install/disconnected_install.adoc

```diff
@@ -352,6 +352,21 @@ $ docker save -o ose3-images.tar \
     registry.access.redhat.com/openshift3/efs-provisioner \
     registry.access.redhat.com/openshift3/node \
     registry.access.redhat.com/openshift3/openvswitch
+    registry.access.redhat.com/openshift3/prometheus
+    registry.access.redhat.com/openshift3/prometheus-alert-buffer
+    registry.access.redhat.com/openshift3/prometheus-alertmanager
+    registry.access.redhat.com/openshift3/prometheus-node-exporter
+    registry.access.redhat.com/cloudforms46/cfme-openshift-postgresql
+    registry.access.redhat.com/cloudforms46/cfme-openshift-memcached
+    registry.access.redhat.com/cloudforms46/cfme-openshift-app-ui
+    registry.access.redhat.com/cloudforms46/cfme-openshift-app
+    registry.access.redhat.com/cloudforms46/cfme-openshift-embedded-ansible
+    registry.access.redhat.com/cloudforms46/cfme-openshift-httpd
+    registry.access.redhat.com/cloudforms46/cfme-httpd-configmap-generator
+    registry.access.redhat.com/rhgs3/rhgs-server-rhel7
+    registry.access.redhat.com/rhgs3/rhgs-volmanager-rhel7
+    registry.access.redhat.com/rhgs3/rhgs-gluster-block-prov-rhel7
+    registry.access.redhat.com/rhgs3/rhgs-s3-server-rhel7
```

Comment:
For multiple lines bash command, a back-slant is needed in the end of each line. (The last one doesn't need it)